.github/workflows/release-semantic-automated-npm.yml

```
@@ -0,0 +1,70 @@
+name: Semantic Release
```

Sorry, I changed my mind about the workflow name. We should not reference a third-party tool, IMO, because we might it in the future. What about "Automatic release"?
```suggestion
name: Automatic release
```

If you agree, please also update the file names and references accordingly (i.e., `.github/workflows/automatic-release.yml`). Thanks!

---

I like, done :+1: